silq/pulses/pulse_sequences.py

```diff
@@ -1168,41 +1168,35 @@ def generate(self):
 
 
 class NMRCPMGPulseSequence(NMRPulseSequence):
-    """`PulseSequenceGenerator` for nuclear magnetic resonance (NMR).
+    """`PulseSequenceGenerator` for nuclear magnetic resonance CP sequences.
 
-    This pulse sequence can handle many of the basic pulse sequences involving
-    NMR. The pulse sequence is generated from its pulse settings attributes.
+    This pulse sequence handles pulse sequences that involve CP experiments,
+    where a train of refocusing pulses is applied after preparing the spin state along the
+    y axis. The main difference between this pulse sequence and a normal
+    NMRPulseSequence sequence is that this pulse sequence accounts for the difference in
+    interdelays between the first pi/2-pi pulses and the refocusing pulses.
+
```

Comment:
I think it's worth explaining why in 1 sentence. The reason for the difference in timing is to have an identical total free-precession time tau for both CP and CPMG sequences, allowing you to directly compare the resulting T2.